pkg/flowaggregator/kafka/protobuf/antrea.proto

```diff
@@ -0,0 +1,66 @@
+// Copyright 2021 VMware, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package antrea_io.antrea.pkg.flowaggregator.kafka.protobuf;
+
+option go_package = "pkg/flowaggregator/kafka/protobuf";
+
+// Antrea proto schema is compatible with go-ipfix Kafka consumer.
+message AntreaFlowMsg {
```

Comment:
Compared to ipfix message, antrea proto schema does not have many fields like network policy rule action or aggregated stats from source/destination node. Are those fields in scope with current PR?

Reply:
Yes, I kept the proto simple, to begin with. Other fields can be added later.